### PR TITLE
Fix/25  template follow design rojak

### PR DESCRIPTION
**Pull request by Codefeak**
Changed some CSS to follow design template n some changes

### CODE REVIEW

**Comment by trungh13** [comment]
merge from develop now first Rojak

File: src/components/Dashboard/DashboardComponent.css [hunk 1]
```diff
@@ -1,44 +1,43 @@
...
-.dashboard-component-header {
+.dashboard-component-footer {
+  min-height: 40px;
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding-bottom: 7.5px;
-  border-bottom: 1px dashed lightgray;
+  padding: 0 20px;
+  background-color:var(--secondary-dark);
```
Comment (trungh13) [inline src/components/Dashboard/DashboardComponent.css]: try secondary-light

File: src/components/Dashboard/StudySync/index.js [hunk 2]
```diff
@@ -21,20 +21,20 @@ class StudySync extends Component {
       <div className="col s6">
         {studysyncs.length !== undefined
```
[the remark this answers is not in the document]
Comment (Codefeak) [reply src/components/Dashboard/StudySync/index.js]: Solved... removed that line ...

File: src/components/Dashboard/Assignment/EachAssignment.js [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
 
 const EachAssignment = ({ assignment }) => (
-  <div className="card-panel hoverable">
-    <div className = "bold">{assignment.titleOfAssignment}</div>
-    <div>{assignment.description}</div>
-    <div>{assignment.date}</div>
+  <div className="row-event">
+    <time className="col-time icon uppercase">
+      <strong>{dayjs(assignment.date.slice(0, assignment.date.length - 7)).format('MMM')}</strong>
+      <span>{dayjs(assignment.date.slice(0, assignment.date.length - 7)).format('DD')}</span>
```
[the remark this answers is not in the document]
Comment (Codefeak) [reply src/components/Dashboard/Assignment/EachAssignment.js]: Solved... slice removed...

File: src/components/Dashboard/StudySync/index.js [hunk 4]
```diff
@@ -21,20 +21,20 @@ class StudySync extends Component {
...
-              </div>
-              <div className="row">
-                <p className="col s8 capitalize">{item.description}</p>
-                <p className="col s4">{`${item.firstName} ${item.lastName}`}</p>
-              </div>
+            <div className="row-studysync" key={`${i + 1}+${item.date}`}>
+                <div className="col-time" >
+                  <time className="icon uppercase">
+                    <strong>{dayjs(item.date.slice(0, item.date.length - 7)).format('MMM')}</strong>
+                    <span>{dayjs(item.date.slice(0, item.date.length - 7)).format('DD')}</span>
```
[the remark this answers is not in the document]
Comment (Codefeak) [reply src/components/Dashboard/StudySync/index.js]: Solved... slice removed...

**Pull request merged**